manifests/application.pp

```diff
@@ -72,6 +73,10 @@
     config_file => "${foreman::plugin_config_dir}/katello.yaml",
   }
 
+  package { "${$scl_postgresql}-postgresql-evr":
```

Comment:
You don't need this if it's a remote database. This also breaks on EL8. You should really pass a variable `$postgresql_evr_package` and determine the full name in `params.pp` where you consider the OS version.

Reply:
This package is only build for EL7, so it looks like I will just do nothing if we're on EL8